src/inverseRobot.js

```diff
@@ -24,7 +24,29 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let counter = 0;
+  const invers = {};
+  const keys = Object.keys(robot);
+  const val = Object.values(robot);
+
+  for (let i = 0; i < val.length; i++) {
+    for (let x = 0; x < val.length; x++) {
+      if (val[i] === val[x]) {
+        counter++;
+      }
+    }
+
+    if (counter > 1) {
+      return null;
+    }
+    counter = 0;
+  }
```

Comment:
nested loop is evil
checklist:
```
2.[CODE STYLE] - use correct check for property presence in object. Some properties can be present, but still contain falsy value.
GOOD EXAMPLE:
if (key in robot) {

EVEN BETTER EXAMPLE:
if (robot.hasOwnProperty(key)) {
```
use this constructions to return `null` where you iterate over the keys